test/unit/gateways/aafes_test.rb

```diff
@@ -0,0 +1,186 @@
+require 'test_helper'
+
+class AafesTest < Test::Unit::TestCase
+  def setup
+    @gateway = AafesGateway.new(identity_uuid: 'identity_uuid')
+
+    # Amount field must be passed in as a decimal
+    @amount = '%.2f' % 100
+    @metadata = {
+      zip: 75236,
+      expiration: 2210
+    }
+
+    @milstar_token = ActiveMerchant::Billing::PaymentToken.new(
+      '900PRPYIGCWDS4O2615',
+      @metadata
+    )
+
+    # TODO: The RRN needs to be unique everytime - the RRN needs to be a base-64 12 character long string
+    @options = {
+      order_id: 'ONP3951033',
+      billing_address: address,
+      description: 'SALE',
+      plan_number: 10001,
+      transaction_id: 6750,
+      rrn: 'RRNPG1685262',
+      term_id: 20,
+      customer_id: 45017632990
+    }
+
+    @bad_options = {
+      order_id: 'ONP3951033',
+      billing_address: address,
+      description: 'Store Purchase',
+      plan_number: 10001,
+      transaction_id: 6750,
+      rrn: 'lengthofrrnmustbe12char',
+      term_id: 20,
+      customer_id: 45017632990
+    }
+  end
+
+  def test_successful_authorize_with_milstar_token
+    @gateway.expects(:ssl_post).returns(successful_authorize_response)
+
+    response = @gateway.authorize(@amount, @milstar_token, @options)
+    assert_success response
+    assert_equal 'Approved', response.message
+    assert response.test?
+  end
+
+  def test_failed_authorize
+    @gateway.expects(:ssl_post).returns(failed_authorize_response)
+
+    response = @gateway.authorize(@amount, @milstar_token, @options)
+    assert_failure response
+    assert_equal 'Decline', response.message
+    assert response.test?
+  end
+
+  # def test_successful_capture
+  # end
+
+  # def test_failed_capture
+  # end
+
+  # def test_successful_refund
+  # end
+
+  # def test_failed_refund
+  # end
+
+  # def test_successful_void
+  # end
+
+  # def test_failed_void
+  # end
+
+  # def test_successful_verify
+  # end
+
+  # def test_successful_verify_with_failed_void
+  # end
+
+  # def test_failed_verify
+  # end
+
+  # def test_scrub
+  #   # assert @gateway.supports_scrubbing?
+  #   # assert_equal @gateway.scrub(pre_scrubbed), post_scrubbed
+  # end
+
+  private
+
+  def pre_scrubbed
```

Comment:
Is this needed? Based on what I know for AAFES, we won't need to scrub the transcript, so I'm not sure this will even be needed down the road.